projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html

```diff
@@ -0,0 +1,78 @@
+    <div fxFlex>
+        <geoengine-layer-selection [types]="inputTypes" formControlName="layer"></geoengine-layer-selection>
+        <geoengine-dialog-section-heading title="Config" subtitle="Specify the output data type"></geoengine-dialog-section-heading>
+
+        <div formGroupName="scale">
+            <mat-slide-toggle formControlName="byKey" #byKey>Use scale factor from metadata key</mat-slide-toggle>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale metadata <i>domain</i></mat-label>
+                <input matInput type="text" formControlName="domain" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale metadata <i>key</i></mat-label>
+                <input matInput type="text" formControlName="key" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale <i>Constant</i></mat-label>
+                <input matInput type="number" formControlName="constant" />
+            </mat-form-field>
+        </div>
+
+        <div formGroupName="offset">
+            <mat-slide-toggle formControlName="byKey" #byKey>Use offset value from metadata key</mat-slide-toggle>
+            <mat-form-field appearance="fill">
+                <mat-label>Offset metadata <i>domain</i></mat-label>
+                <input matInput type="text" formControlName="domain" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Offset metadata <i>key</i></mat-label>
+                <input matInput type="text" formControlName="key" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Offset <i>Constant</i></mat-label>
+                <input matInput type="number" formControlName="constant" />
+            </mat-form-field>
+        </div>
+
+        <mat-form-field appearance="fill">
+            <mat-label>Scale Type</mat-label>
+            <mat-select formControlName="scaleType">
+                <mat-option *ngFor="let scaleType of scaleTypes" [value]="scaleType.type">
+                    {{ scaleType.type }} <i>{{ scaleType.formular }}</i>
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+
+        <geoengine-operator-output-name
+            formControlName="name"
+            [suggestion]="
+                (form.controls.layer.valueChanges | async)?.name + ' ' + (form.controls.scaleType.valueChanges | async)?.toString()
+            "
+        >
+            <mat-hint align="end" class="error" *ngIf="form.controls.name.errors?.required || form.controls.name.errors?.onlyWhitespace">
+                The name must be non-empty.
+            </mat-hint>
+        </geoengine-operator-output-name>
+    </div>
+    <div class="actions">
+        <button type="submit" mat-raised-button color="primary" [disabled]="disallowSubmit | async">Create</button>
```

Comment:
should be disabled if no scale type is selected

Reply:
[1a42aea](https://github.com/geo-engine/geoengine-ui/pull/471/commits/1a42aeabc88bf4170889625b2bbe12e1eff39685)